api/BUILD

```diff
@@ -1,5 +1,17 @@
 load("//bazel:api_build_system.bzl", "api_proto_library")
 
+cc_library(
```

Comment:
I've switched to avoiding this. It was a convenience to avoid having to do `bind` for each of the libraries in the main Envoy repo, but it doesn't matter, we have to do something either side so it's fine to keep it fine grained, as you point out it's the Bazel way.